app/presenters/account_show_presenter.rb

```diff
@@ -34,10 +36,44 @@ def show_gpo_partial?
     user.gpo_verification_pending_profile?
   end
 
+  def show_ipp_partial?
```

Comment:
There's a lot of new public methods being added, but only one of them has new test coverage in `app/presenters/account_show_presenter_spec.rb`. I think some of these could be private methods, but the rest I'd expect test coverage.